charts/zenko/Chart.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
 description: A Helm chart for Kubernetes
 name: zenko
-version: 1.0.0
+version: 1.0.0-RC5
```

Comment:
would it still be 1.0.0 if it's RC5?

Reply:
Yeah it's supposed to be our 5th release candidate of our 1.0.0. (Although it doesn't really make sense because we're on development/0.9)

Reply:
ty for clarifying

Reply:
Yeah our versioning isn't 100% consistent but we're trying to fix that as we go